Headers/Public/VLCMediaListPlayer.h

```diff
@@ -36,6 +36,13 @@ typedef NS_ENUM(NSInteger, VLCRepeatMode) {
     VLCRepeatAllItems
 };
 
+@protocol VLCMediaListPlayerDelegate <NSObject>
+@optional
+- (void)mediaListPlayerPlayed:(VLCMediaListPlayer *)player;
```

Comment:
startedPlayback? or is this the callback for being done with playback. 
comments might help especially for other users of the sdk